content/blog/2025/anchor-position-area.md

```diff
@@ -0,0 +1,209 @@
+The examples in this article are best viewed in a Chromium browser, version 131
+or later.
+
+{% endcallout %}
+
+On the surface, `position-area` seems like just a shorthand.
+Just replace `top: anchor(bottom); left: anchor(right)`
+with `position-area: bottom right` and it works.
+
+Or go even further, and replace that with `position-area: end`.
+
+{{ embed.codepen(
+  id='zxOVXOm',
+  title='position-area containing block',
+  user='jamessw',
+  height=300,
+  tab='css,result'
+) }}
+
+The idea behind `position-area` is that in most cases, you want your positioned
+element to be touching a specific corner or edge of the anchor. `position-area`
+creates a grid around the anchor, and lets you place your positioned element
+within that grid.
+
+[Anchor-tool.com](https://anchor-tool.com/) is a fantastic tool for exploring
+the different values for positioning an element within that grid.
+
+[{{ embed.img(
+  src='blog/2025/anchor-tool-com.jpg',
+  alt='A screenshot of anchor-tool.com, demonstrating an element positioned with position-area: top span-left.',
+  attrs={
+    'class': 'align-center'
+  }
+) }}](https://anchor-tool.com/)
+
+But `position-area` is more than a shorthand for `anchor()`, introducing some
+behaviors that may be surprising.
+
+## A new containing block
+
+One of the key differences between using `anchor()` and `position-area` is that
+`position-area` creates a new containing block for the positioned element.
+
+If you are using `anchor()`, the positioned element's containing block works
+like normal. For a `fixed` position element, it will usually be the
+viewport. For an `absolute` positioned element, it will usually be the
+closest ancestor that has a `position` besides `static`. The `anchor()` function
+then resolves to a length that positions the element as specified, within that
+containing block.
+
+`position-area` creates a containing block that does not map to an element.
+Instead, it is defined by the relationship between the anchor and the anchor's
+containing block.
+
+This containing block is generated by the grid area that you select using
+`position-area`. For instance, with `position-area: top left`, the new
+containing block starts at the top left corner of the anchor's containing block
+and extends to the top left corner of the anchor itself.
+
+{{ embed.img(
+  src='blog/2025/anchor-area-containing-block.png',
+  alt='A 9-space grid, like a tic-tac-toe board, with an Anchor in the center square, and a square within the top left square for a new containing block.',
+  attrs={
+    'class': 'align-center',
+    'width': '1089',
+    'height': '997'
+  }
+) }}
+
+Note that the element we're positioning doesn't have to fill this entire area,
+but we're choosing the container that we want to position our element inside of.
+
+## Positioning within the containing block
+
+Using `position-area` also handles some defaults so that the positioned element
+is placed near to the anchor, within its new containing block.
```

Comment:
Do we need the word "to" here? My brain kept wanting to read this as ... 

is placed near the anchor, within ...

Reply:
ah, looked it up after I read the "close to" in a few lines

https://dictionary.cambridge.org/us/grammar/british-grammar/near-and-near-to